.codecov.yml

```diff
@@ -0,0 +1,13 @@
+codecov:
+  require_ci_to_pass: yes
+
+coverage:
+  precision: 2
+  round: down
+  range: "70...100"
+  status:
+    project:
+      default:
+        target: 70%    # the required coverage value
```

Comment:
Do we have an idea what our current coverage is?  Ideally this should be set just below what our current coverage is (so it doesn't get any worse) and then as we incrementally improve we can raise the bar.

Also I believe we can set `auto` here which will measure PR coverage against the branch coverage it's being committed against, thus always improving coverage.

Reply:
> Do we have an idea what our current coverage is?

Our current code coverage is 100% after I use jacoco plugin to do `./gradlew jacocoTestReport` locally.

Reply:
> > Do we have an idea what our current coverage is?
> 
> Our current code coverage is 72.77%.

Nice! That output will help us figure out where we need more.  As said "auto" effectively makes the target 72.77%.